lib/LoginStrategy.ts

```diff
@@ -239,4 +254,78 @@ export class LoginStrategy extends BaseStrategy {
       return acc
     }, [] as Claim[])
   }
-}
+
+  /**
+   * @description checks if the claimer is authenticated into the DID Document
+   *
+   * @param token
+   * @param payload
+   *
+   * @returns {string} issuer DID or null
+   */
+  async verifyClaim(token: string, payload: ITokenPayload): Promise<string> {
```

Comment:
```suggestion
  async verifyAuthenticationClaim(token: string, issuerDID: string): Promise<string> {
```
To me this is a clearer function signature as it's not the entire `ITokenPayload` that we need, it is just the `payload.iss` so better to no force calls to provide an entire `ITokenPayload`.
If this change, is made then the description of the function could be changed to "checks a token was signed by the issuer DID or a valid authentication delegate of the issuer"